spec/yaml/swiftnav/sbp/logging.yaml

```diff
@@ -12,20 +12,33 @@ package: swiftnav.sbp.logging
 description: |
   Logging and debugging messages from the device. These are in the
   implementation-defined range (0x0000-0x00FF).
-stable: False
+stable: True
```

Comment:
The stability will not apply to deprecated messages. I need to update that here and in a couple of other places. But, yes, `MSG_TWEET` will be stable.